engines/hopkins/events.cpp

```diff
@@ -251,9 +253,11 @@ void EventsManager::pollEvents() {
 		case Common::EVENT_RETURN_TO_LAUNCHER:
 			return;
 
+		case Common::EVENT_CUSTOM_ENGINE_ACTION_START:
+			handleKey(event);
+			return;
```

Comment:
Since this doesn't set `_keyState`, will it work correctly with `EventsManager::waitKeyPress()` below, when the user maps an ASCII key to an action (e.g. the 'w' key)?

Reply:
I've re-read the whole event handling code in this engine. There are three separate cases where keys are handled:
- `DialogsManager::testDialogOpening()`, which handles the inventory, save/load screens etc, and it sets `_gameKey`. This is what is modified by this PR
- `ComputerManager::showComputer()`, which handles in-game computers. Reads only numeric keys via `waitKeyPress()`, and does not set `_gameKey`. This is not affected by this PR
- `ComputerManager::displayMessage()`, which is used for text input in in-game computers. Not affected, either
- `ComputerManager::readText()`, which displays texts for a menu entry and clears text input. Not affected, either
- `ComputerManager::getScoreName()`, which gets text input for the highscores of the breakout mini-game. Not affected, either

Thus, this change related to the keymapper is correct, as this part of the code is only used in menus, and the rest of the text input code is not affected